### ISSUE TITLE
MPEG Surround test-vector has incorrect duration of audio track

### ISSUE THREAD

**Issue by KozhinM** [Dash-Industry-Forum/Test-Vectors#18]
MPD:
http://dash.edgesuite.net/dash264/TestCasesMCA/fraunhofer/MPEG_Surround/1/6chID/6chId_480p_single_adapt_mps5_1_sidx.mpd

Details:
This mpd has `@mediaPresentationDuration = 11.5416666667`
After the segments have been appended to the SourceBuffer the video buffer level is exactly the same value as an mpd duration, but audio level is only `11.3066665`. Thus, audio track is by `0.235sec` shorter than an expected value.

**Comment by waqarz** [comment]
Including some relevant people in the issue.

If I am not wrong then we can expect a difference of approx. a sample duration between the presentation duration and actual track duration. 235 msec may be too much, but I think this is still not a DASH conformance issue.

**Comment by KozhinM** [comment]
@waqarz, do you know the value of an acceptable difference or how it is calculated? I have not found it, so any info is really helpful. Right now dash.js player expects an exact match between declared and actual durations. If some difference is allowed then the player will be fixed, but if 0.235 exceeds the threshold I think this should be considered as an mpd issue, not the player.

**Comment by waqarz** [comment]
@KozhinM I agree that 0.235 sec difference is a content authoring issue at least. But we should seek some expert opinion from DASH-IF on this issue if you agree.

**Comment by KozhinM** [comment]
Yes, expert's opinion would be very helpful.
